openmdao/solvers/linear/direct.py

```diff
@@ -285,7 +280,7 @@ def _linearize(self):
                     if 'exactly singular' in str(err):
                         raise RuntimeError(format_singular_csc_error(system, matrix))
                     else:
-                        reraise(*sys.exc_info())
+                        raise sys.exc_info()
```

Comment:
`raise sys.exc_info()` isn't valid, at least not in python 3.8.

Reply:
Fixed